raiden-ts/tests/e2e/e2e.spec.ts

```diff
@@ -55,24 +55,24 @@ async function createRaiden(account: number | string | Signer): Promise<Raiden>
 }
 
 const wait = async (timeInMs: number): Promise<void> =>
-  new Promise((resolve) => setTimeout(() => resolve(), timeInMs));
```

Comment:
Nitpick: these parenthesis changes are against our current linter rules. Do you have up-to-date packages?